### DISCUSSION TITLE
Requesting a new ckan-harvest release

### DISCUSSION THREAD

**Discussion by ricardogsilva** [ckan/ckan#6666] [General]
(Sorry if this is the wrong place to pose this question - kindly direct me elsewhere if so)

While starting to play around with the ckan-harvest plugin, I tried running `ckan harvester run-test <my-source-id>` and got an error that comes down to

```python
UnboundLocalError: local variable 'force_import_val' referenced before assignment
```

Checking the [ckanext-harvest](https://github.com/ckan/ckanext-harvest) repo I see that this has been fixed in ckan/ckanext-harvest@254ffb065a775ab35c568243bb1d1cede12f7863

Additionally, there seem to have been a number of bugfix commits to ckanext-harvest since the release of v1.3.3 (AKA the latest released version), which was about six months ago.

With this in mind, I am kindly asking for a new release of ckan-harvest, in order to have a clean way to benefit from these bugfixes. Is this something that the CKAN dev team is willing to consider? Is there something that I may help with in order to facilitate the release process?

Thanks in advance

**Comment by amercader** [comment]
Thanks @ricardogsilva  and sorry it took a while. [v1.3.4](https://github.com/ckan/ckanext-harvest/releases/tag/v1.3.4) is out now.

Help reviewing incoming PRs is greatly appreciated, so if experienced users want to become maintainers feel free to come forward!
